fix: expand the breadcrumb when [[link]], don't navigate

juniusfree opened the pull request:
**Current behavior:** See https://github.com/athensresearch/athens/pull/972#issuecomment-820496888
**Expected behavior:** When clicking the parsed block reference or page reference on the breadcrumb, Athens should navigate to the block page only
**Solution**: Add an invisible element above the parsed content to prevent it from getting clicked.

![athens-972-comment](https://user-images.githubusercontent.com/8164667/115030305-87f5a480-9ef9-11eb-8b39-6898b14ebe71.gif)

tangjeff0 commented:
What if there was just one `span`?

What if the one `span` just did `(.. e preventDefault)` `:on-click`?

juniusfree commented:
@tangjeff0 I can't make it work using `preventDefault` but I updated it with one `span` using `pointer-events`. It still works the same. I added the `relative` property because the `opacity` in hashtag and page-link components are messing up with `pointer-events`.

tangjeff0 commented:
Great work @juniusfree  :)
